example/requirements.txt

```diff
@@ -0,0 +1,2 @@
+django-select2
```

Comment:
Can we replace that line by something like `-e ..  # django-select2` so that the local copy is used and changes to the code will be reflected live?